bin/dart_openapi_codegen.dart

```diff
@@ -637,8 +686,9 @@ class Operation {
       '{\n' +
       parameters.entries
           .where((e) => e.value.type == ParameterType.query)
-          .map((e) =>
-              '      ${e.value.schema.dartToQueryEntry(e.key, variableName(e.key))},\n')
+          .map((e) => e.value.explode
+              ? '${e.value.schema.dartCondition(variableName(e.key))}  ${e.value.schema.dartType.startsWith('List') ? 'for (final query in ${variableName(e.key)}) \'${e.key}\': query' : '...${variableName(e.key)}'}'
```

Comment:
this is wrong, imo. If you explode a list, you still need to pass the values as a list to the query parameters of the request, since that is how dart works. Also maybe transform this into a switch or so, nested ternaries are hard to read.